dpnp/backend/extensions/lapack/lapack_py.cpp

```diff
@@ -40,6 +42,7 @@ namespace py = pybind11;
 void init_dispatch_vectors(void)
 {
     lapack_ext::init_syevd_dispatch_vector();
+    lapack_ext::init_gesv_dispatch_vector();
```

Comment:
to keep the lexicographical order
```suggestion
    lapack_ext::init_gesv_dispatch_vector();
    lapack_ext::init_syevd_dispatch_vector();
```